### ISSUE TITLE
Warning running Liberty with Java 9

### ISSUE THREAD

**Issue by donbourne** [OpenLiberty/open-liberty#2328]
Reported by Alasdair...

just accidentally ran liberty with Java 9 and got this:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.ibm.ws.logging.internal.impl.IntrospectionLevelMember$2 (file:/Users/nottinga/Documents/github/cloud-native-examples/serviceA/target/liberty/wlp/lib/com.ibm.ws.logging_1.0.20.jar) to field java.util.concurrent.atomic.AtomicReference.serialVersionUID
WARNING: Please consider reporting this to the maintainers of com.ibm.ws.logging.internal.impl.IntrospectionLevelMember$2
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

We need to clean up the code so that it doesn't cause this warning.

**Comment by NottyCode** [comment]
This is the Java 9 vm I have installed: java version "9.0.1"
Java(TM) SE Runtime Environment (build 9.0.1+11)
Java HotSpot(TM) 64-Bit Server VM (build 9.0.1+11, mixed mode)

I was expecting it to use Java 8, but I'm pretty sure those warnings only come out in Java 9, so it must be this one.

**Comment by alexmulholland** [comment]
I just saw the same issue running 19.0.0.2 with OpenJDK 11.  Since Java 11 is an LTS release we'll be moving products to it so it would be nice to get this fixed :-)  I haven't tried 19.0.0.3 though.

Launching server1 (Open Liberty 19.0.0.2/wlp-1.0.25.cl190220190222-1311) on OpenJDK 64-Bit Server VM, version 11.0.2+9 (en_US)
[AUDIT   ] CWWKE0001I: The server server1 has been launched.
[AUDIT   ] CWWKZ0058I: Monitoring dropins for applications.
[AUDIT   ] CWWKT0016I: Web application available (default_host): http://localhost:9080/sofydemo/
[AUDIT   ] CWWKZ0001I: Application sofydemo started in 1.610 seconds.
[AUDIT   ] CWWKF0012I: The server installed the following features: [jsp-2.3, servlet-4.0, localConnector-1.0, jsonp-1.1, jaxrsClient-2.1, el-3.0].
[AUDIT   ] CWWKF0011I: The server server1 is ready to run a smarter planet.
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.ibm.ws.logging.internal.impl.IntrospectionLevelMember$2 (file:/Users/alexmulholland-macbook/eclipse-workspace/ol-19002/lib/com.ibm.ws.logging_1.0.25.jar) to field java.util.concurrent.atomic.AtomicReference.serialVersionUID
WARNING: Please consider reporting this to the maintainers of com.ibm.ws.logging.internal.impl.IntrospectionLevelMember$2
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

**Comment by donbourne** [comment]
thanks @alexmulholland -- we haven't fixed this yet, so probably no point trying in 19.0.0.3.  I've tagged it so it will get picked up in our iteration planning.
